app.rb

```diff
@@ -35,15 +62,38 @@ def list_all_games
 
   def list_all_genres; end
 
-  def list_all_labels; end
+  def list_all_labels
```

Comment:
@Chimwemwe-127001  Nice implementing List labels but I think you forgot to implement add_label so we can add a label

Reply:
Thanks for the insightful review.
I've implemented changes as requested